doc/api.odocl

```diff
@@ -77,6 +77,7 @@ Pkcs11_bindings
 Pkcs11_data
 Pkcs11_generator
 Pkcs11_helpers
+Pkcs11_high_level
```

Comment:
Can you rename it `P11_driver`? I think that it reflects better what this module does.